Mathlib/Computability/DFA.lean

```diff
@@ -92,7 +92,7 @@ theorem evalFrom_of_append (start : σ) (x y : List α) :
 #align DFA.eval_from_of_append DFA.evalFrom_of_append
 
 /-- `M.accepts` is the language of `x` such that `M.eval x` is an accept state. -/
-def accepts : Language α := fun x => M.eval x ∈ M.accept
+def accepts : Language α := {x | M.eval x ∈ M.accept}
```

Comment:
This is still a defeq abuse because `Language` and `Set` are not interchangeable.
https://github.com/leanprover-community/mathlib/pull/18451 had a path to resolve this, but was more trouble that I cared to deal with.

Having said that, this change is certainly not harmful, so fine to leave it

Reply:
They are not interchangeable but `Language` is defined as `Set`, not `_ → Prop`, so this is a step in the right direction. The proper fix would be to introduce `Language.mk`.

Reply:
`Set.up` is pretty much exactly that function, as `SetSemiring` has all the same instances as `Language`.